src/Samples/GtkSource-5/SourceView/SourceViewTest.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Can you please rename your project into something like "GtkSourceView" "SimpleGtkSourceView". Please avoid the suffix "Test" as this would look like it is a unit test project.